addons/html_builder/static/src/plugins/image/image_shape_option.js

```diff
@@ -1,9 +1,9 @@
 import { BaseOptionComponent, useDomState } from "@html_builder/core/utils";
 import { toRatio } from "@html_builder/utils/utils";
-import { ShapeSelector } from "../shape/shape_selector";
+import { ShapeSelector } from "@html_builder/plugins/shape/shape_selector";
 
 export class ImageShapeOption extends BaseOptionComponent {
-    static template = "website.ImageShapeOption";
+    static template = "html_builder.ImageShapeOption";
```

Comment:
It's just a question, I have the impression that shapes don't exist in the old mass_mailing builder. Is this an imp?